e2e/scheduler_test.go

```diff
@@ -28,21 +26,26 @@ func testScheduler() {
 				return err
 			}
 
-			if len(podlist.Items) != 1 {
+			if len(podlist.Items) == 0 {
 				return errors.New("pod is not found")
 			}
 
-			pod := podlist.Items[0]
-			for _, cond := range pod.Status.Conditions {
-				fmt.Println(cond)
-				if cond.Type == corev1.PodReady && cond.Status == corev1.ConditionTrue {
-					return nil
+			for _, pod := range podlist.Items {
+				podReady := false
+				for _, cond := range pod.Status.Conditions {
+					fmt.Println(cond)
```

Comment:
for debug?

Reply:
I think so, this line was already in place before the change.
And changed it to use `GinkgoWriter`.